docs/farming-&-staking/farming/additional-guides/gpu-plotter.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Experimental GPU Plotter
+sidebar_position: 4
+description: Tips on GPU Plotter
+keywords:
+    - Plotter
+    - Farming
+    - GPU
+---
+
+The first test builds of the **GPU Plotter** are now available!!!
+
+[Download GPU Plotter](https://github.com/autonomys/subspace/actions/runs/10850781628)
+
+## Overview
+The GPU Plotter will eventually support most NVIDIA and AMD video cards. For now, only NVIDIA cards are supported.
```

Comment:
I think we should describe the overall idea of the GPU plotter first. In what cases one might consider using them, and what are the benefits.